nekoyume/Assets/_Scripts/UI/HeaderMenu.cs

```diff
@@ -144,6 +144,25 @@ public override void Initialize()
 
             menuToggleDropdown.onValueChanged.AddListener((value) =>
             {
+                if (value)
+                {
+                    CloseWidget = () => { menuToggleDropdown.isOn = false; };
+                    WidgetStack.Push(gameObject);
+                }
+                else
+                {
+                    CloseWidget = null;
+                    Observable.NextFrame().Subscribe(_ =>
+                    {
+                        var list = WidgetStack.ToList();
+                        list.Remove(gameObject);
```

Comment:
Is this widget clearly removed in `WidgetStack` this line?

Reply:
@Unengine Yes. I had test that. Is there anything might be a problem?

Reply:
I just worried about converting stack to list and modifying the value causes no problem. If it is working right, i think it is ok.